easybuild/easyconfigs/i/isoCirc/isoCirc-1.0.4-foss-2020b.eb

```diff
@@ -0,0 +1,71 @@
+easyblock = 'PythonBundle'
+
+name = 'isoCirc'
+version = '1.0.4'
+
+homepage = 'https://github.com/Xinglab/isoCirc'
+description = """isoCirc: computational pipeline to identify high-confidence
+BSJs and full-length circRNA isoforms from isoCirc long-read data
+"""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ('Biopython', '1.78'),
+    ('SciPy-bundle', '2020.11'),
+    ('matplotlib', '3.3.3'),
+    ('Pysam', '0.16.0.1'),
+    ('pybedtools', '0.8.2'),
+    ('minimap2', '2.18'),
+]
+
+use_pip = True
+download_dep_fail = True
+sanity_pip_check = True
+
+exts_list = [
+    ('argcomplete', '1.12.3', {
+        'checksums': ['2c7dbffd8c045ea534921e63b0be6fe65e88599990d8dc408ac8c542b72a5445'],
+    }),
+    ('argh', '0.26.2', {
+        'checksums': ['e9535b8c84dc9571a48999094fda7f33e63c3f1b74f3e5f3ac0105a58405bb65'],
+    }),
+    ('crlibm', '1.0.3', {
+        'checksums': ['48e17981f90d69c6bb0013f68bacbe7a157de864a533d15dd196ca7e98348a35'],
+    }),
+    ('gffutils', '0.10.1', {
+        'checksums': ['a8fc39006d7aa353147238160640e2210b168f7849cb99896be3fc9441e351cb'],
+    }),
+    ('mappy', '2.20', {
+        'checksums': ['00c8720783ac982b514b01b86349ede9809eeaccdf1328acb5a16dfeef51458b'],
+    }),
+    ('pyfaidx', '0.6.2', {
+        'checksums': ['d1258f8d053cba0c90fe329254e8ec59eb28b535b48d9d06e8c7f1d74b8e4531'],
+    }),
+    ('pyinterval', '1.2.0', {
+        'modulename': 'interval',
+        'checksums': ['8c46224a05815affa803ed5620432236bba620a246701b860d98ae7358a41d21'],
+    }),
+    (name, version, {
+        'source_tmpl': 'v%(version)s.tar.gz',
+        'source_urls': ['https://github.com/Xinglab/isoCirc/archive'],
```

Comment:
@zarybnicky Makes more sense to use the source tarball from PyPI? Cfr. https://pypi.org/project/isocirc

Reply:
@boegel The PyPI archive does work, but it doesn't contain the example data that the GitHub does that are used in the sanity check. I'll keep using the GitHub archive unless that's what you want

Reply:
OK, in that case it makes sense to use the source tarball from GitHub, thanks for clarifying.